Sources/CMakeLists.txt

```diff
@@ -14,12 +14,12 @@
 
 add_subdirectory(CoreFoundation)
 add_subdirectory(_CFXMLInterface)
-if(NOT CMAKE_SYSTEM_NAME STREQUAL "WASI")
+if(FOUNDATION_BUILD_NETWORKING)
     add_subdirectory(_CFURLSessionInterface)
 endif()
 add_subdirectory(Foundation)
 add_subdirectory(FoundationXML)
-if(NOT CMAKE_SYSTEM_NAME STREQUAL "WASI")
+if(FOUNDATION_BUILD_NETWORKING)
```

Comment:
Should we make an equivalent change to the Package manifest to not build `FoundationNetworking` targets for WASI?

Reply:
SwiftPM does not have a way to define a target conditionally based on the target triple, so developers need to specify `--target` or `--product` explicitly instead of excluding them in Package.swift manifest.